Cubical/HITs/SetTruncation/Properties.agda

```diff
@@ -169,3 +173,14 @@ Iso.rightInv setTruncOfProdIso =
   prodElim (λ _ → isOfHLevelPath 2 (isSet× setTruncIsSet setTruncIsSet) _ _) λ _ _ → refl
 Iso.leftInv setTruncOfProdIso =
   elim (λ _ → isOfHLevelPath 2 setTruncIsSet _ _) λ {(a , b) → refl}
+
+moveSetTruncΣ : ∀ {ℓ ℓ'} {A : Type ℓ} {B : A → Type ℓ'} → Iso ∥ Σ A B ∥₂ ∥ Σ A (λ x → ∥ B x ∥₂) ∥₂
```

Comment:
This name is not great, it should contain `Iso`. Also, isn't the levels and A already quantified using variable in this file?